app/experimenter/static/rapid/components/forms/ExperimentForm.tsx

```diff
@@ -285,6 +307,18 @@ export const BranchesForm: React.FC = () => {
                           </div>
                         </div>
                       </div>
+                      {!variant.is_control && (
+                        <div>
+                          <button
+                            aria-label="Close"
+                            className="close"
+                            type="button"
+                            onClick={() => handleRemoveBranch(i)}
+                          >
+                            <span style={{ color: "red" }}>&times;</span>
```

Comment:
Using `red` emphasizes this in the UI, which I don't think is what we want here. Just keep it the default colour